client/src/components/Requests/RequestMenu/RequestMenu.tsx

```diff
@@ -6,35 +6,40 @@ import SettingsIcon from '@material-ui/icons/Settings';
 import React from 'react';
 import Menu from '@material-ui/core/Menu';
 import MenuItem from '@material-ui/core/MenuItem';
+import { useRequest } from '../../../context/useRequestContext';
 
 interface Props {
   request: Request | null | undefined;
 }
 
 const RequestMenu = ({ request }: Props): JSX.Element => {
+  const { sendResponse } = useRequest();
   const classes = useStyles();
-  const accepted = request?.accepted == true;
   const [menuToggle, setmenuToggle] = React.useState<null | HTMLElement>(null);
+  if (request) {
+    const accepted = request.accepted == true;
 
-  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
-    setmenuToggle(event.currentTarget);
-  };
-
-  const handleClose = () => {
-    setmenuToggle(null);
-  };
+    const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+      setmenuToggle(event.currentTarget);
+    };
 
-  if (request) {
+    const handleClose = () => {
+      setmenuToggle(null);
+    };
+    const handleSubmit = (bookingStatus: string) => {
+      sendResponse(bookingStatus, request._id);
+      setmenuToggle(null);
+    };
     return (
       <Box>
         <IconButton aria-controls="simple-menu" aria-haspopup="true" onClick={handleClick} className={classes.button}>
           <SettingsIcon />
         </IconButton>
         <Menu id="request-menu" anchorEl={menuToggle} keepMounted open={Boolean(menuToggle)} onClose={handleClose}>
-          <MenuItem className={classes.menuText} disabled={accepted} onClick={handleClose}>
+          <MenuItem className={classes.menuText} disabled={accepted} onClick={() => handleSubmit('accept')}>
```

Comment:
You could simply use `request.accepted` here

Reply:
Change made.

Reply:
Please see the above related comment